source/development_process/configuring_git.rst

```diff
@@ -0,0 +1,71 @@
+Configuring Git
+===============
+Git's default configuration is very good, but adding a couple lines can make
+working with git much easier.
+
+``.gitconfig``
+--------------
+
+It is imperative that developers set their email
+and user name in their ``~/.gitconfig`` file. Developers may use any email
+address they prefer, including their work email. If developers want commits
+they make using that email address to show up as part of their github commit
+count they should add that email on
+`Github <https://github.com/settings/emails>`_.
+
+When editing commits, git
+defaults to using the nano editor. Most developers will want to change this to
+vim or the editor of their choice. Setting up color configuration can make git
+command line output much easier to read. It can also be handy to set up aliases
+for commonly used or long git sub-commands. Using the example config, running
+``git net`` will run ``git log`` with a series of useful options.
+
+.. code::ini
```

Comment:
Setting

```ini
[push]
    default = nothing # or 'current'
```

Will save a lot of headaches related to pushing branches and accidentally force-pushing the wrong branch and such.